sp/src/game/client/c_basecombatcharacter.h

```diff
@@ -99,7 +99,7 @@ class C_BaseCombatCharacter : public C_BaseFlex
 
 #ifdef GLOWS_ENABLE
 	CGlowObject			*GetGlowObject( void ){ return m_pGlowEffect; }
-	virtual void		GetGlowEffectColor( float *r, float *g, float *b );
+	virtual void		GetGlowEffectColor( float *r, float *g, float *b, float *a );
```

Comment:
It's unlikely, but I wonder if any forks of Mapbase call this function from a different class and would have the signature broken by this change. If we really want to be safe, we could add a default 1.0 value.